CompetitionTemplate/src/competition/subsystems/arm/arm_commands/LowerArm.java

```diff
@@ -0,0 +1,31 @@
+package competition.subsystems.arm.arm_commands;
+import competition.subsystems.arm.ArmSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class LowerArm extends BaseCommand {
+    ArmSubsystem armSubsystem;
+
+    public LowerArm(ArmSubsystem armSubsystem) {
+        this.armSubsystem = armSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        if (armSubsystem.isArmAtMaximumHeight()){
+            end();
+        }else {
+            armSubsystem.armMotor.set(1);
```

Comment:
Only subsytems should get to know about motors, you'll want to call a method on the subsystem to do what you want. Remember this was so that if we ever switch the number of motors we only have to update the subsystem and not all the commands.